README.md

```diff
@@ -1,10 +1,41 @@
-# Infinite Industries 
+# Infinite Industries
+<img src="./docs/doc-splash.png">
 
 ## Mission Statement
+<p>Infinite Industries makes experimental contemporary culture more accessible to everyone. Our digital platform and open-source tools highlight cultural events and experiences so more of us can discover, share and participate in them.
+</p>
 
-## Directory Structure
+<p>
+<b>why the name?</b>
+Artists constantly evolve, inventing new genres and redefining their methods. The processes and mediums of the past aren’t always sufficient for the challenges of the future. We can thank artists for an infinity of possibilities. Production trends organize beneath the umbrella term "industry," creating an infinity of industries and our special name ;)
+</p>
+<p>Our efforts are currently focused on Lexington, Kentucky USA with lots of future plans for world domination.</p>
 
+## Project Components
+The project is split into API and various components that source information from it. We are intentionally trying to centralize the information and de-centralize the types of services that consume the data.
+### API
```

Comment:
Might want to add a new line between end of this paragraph and the next header. It's also typicial in markdown to leave a blank line between the header and the body text so for example

## Foo Bar

Some text

## Much Foo

More text